src/commands/lint.ts

```diff
@@ -73,11 +72,11 @@ export class LintCommand implements Command {
     }
   ];
 
-  run(options, config): Promise<any> {
+  run(options: CommandOptions, config: ProjectConfig): Promise<any> {
     // Defer dependency loading until this specific command is run
     const polylint = require('polylint/lib/cli');
 
-    let lintFiles: string[] = options.input;
+    let lintFiles: string[] = options['input'];
```

Comment:
was this clang or TS, or is this just personal preference? If we're talking personal preference I prefer indexing properties only when necessary.